rasa_nlu/components.py

```diff
@@ -288,6 +328,20 @@ def partially_process(self, message):
                         "to missing pipeline.")
         return message
 
+    @classmethod
+    def can_handle_language(cls, language):
+        # type: (Hashable) -> bool
+        """Check if component support specific language.
+
+        this method can overwrite when needed.
+        (e.g. dynamic determine which language is supported.)"""
```

Comment:
"Check if component supports a specific language. This method can be overwritten when needed. (e.g. dynamically determine which language is supported.)"